client/state/sites/actions.js

```diff
@@ -91,15 +91,14 @@ export function requestSites() {
 			} )
 			.then( ( response ) => {
 				const jetpackCloudSites = response.sites.filter( ( site ) => {
-					// Filter Jetpack Cloud sites to exclude P2 sites by default.
+					// Filter Jetpack Cloud sites to exclude P2 and Simple non-Classic sites by default.
 					const isP2 = site?.options?.is_wpforteams_site;
-					let filterCondition = ! isP2;
-					// Filter out simple sites if feature flag is not enabled.
-					if ( ! isEnabled( 'jetpack/manage-simple-sites' ) ) {
```

Comment:
@lupus2k @candy02058912 I removed the flag check since we are using `site_filter` to control the site types being returned. WDYT?

Reply:
Sounds good to me. As normal users currently don't have a way to turn on `wp-admin` interface.